### PR TITLE
Add initial files

### PR DESCRIPTION
**Pull request by JoeShep**
I made this code rock. I added actual files. Now it's not boring.

### CODE REVIEW

**Comment by zoeames** [comment]
Joseph you should really add something to your files

**Comment by zoeames** [comment]
:+1:

File: main.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+body {
+  background-color: red;
```
Comment (zoeames) [inline main.css:2]: i wish this was blue

Comment (JoeShep) [inline main.css:2]: Ok, if you want me to

**Pull request merged**